test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/schedule/HapiScheduleSign.java

```diff
@@ -76,11 +80,21 @@ public HederaFunctionality type() {
 
 	@Override
 	protected Consumer<TransactionBody.Builder> opBodyDef(HapiApiSpec spec) throws Throwable {
+
 		ScheduleSignTransactionBody opBody = spec
 				.txns()
 				.<ScheduleSignTransactionBody, ScheduleSignTransactionBody.Builder>body(
 						ScheduleSignTransactionBody.class, b -> {
-							b.setScheduleID(asScheduleId(schedule, spec));
+							ScheduleID id;
+							try {
+								id = asScheduleId(schedule, spec);
+							} catch (RegistryNotFound e) {
+								if (ignoreMissing) {
+									return;
```

Comment:
This `return` is concerning. The expected returned type is `Consumer<TransactionBody.Builder>`

Reply:
@qnswirlds  Thank you! Fixed to throw error only when the boolean is not set.